test/unit/specs/fields/fieldNoUiSlider.spec.js

```diff
@@ -36,7 +36,71 @@ describe("fieldNoUiSlider.vue", () => {
 			expect(field.$el).to.be.exist;
 
 			expect(input).to.be.defined;
-			expect(input.classList.contains("slider")).to.be.true;			
+			expect(input.classList.contains("slider")).to.be.true;
+			expect(input.disabled).to.be.undefined;
+		});
+
+		before( () => {
+			vm.$appendTo(document.body);
+		});
+
+		it("should contain an handle element", (done) => {
+			if (window.noUiSlider) {
+				vm.$nextTick( () => {
+					let handle = input.querySelectorAll(".noUi-handle");
+					expect(handle.length).to.be.equal(1);					
+					expect(handle[0]).to.be.defined;
+					// expect(input.classList.contains("noui-target")).to.be.true;
+					done();
+				});
+			} else {
+				this.skip();
+			}
+		});
+
+		it.skip("should contain the value", (done) => {
+			vm.$nextTick( () => {
+				let origin = input.querySelectorAll(".noUi-origin")[0];				
+				expect(origin.style.left).to.be.within("70%", "90%");
+				done();
+			});
+		});
+
+		before( () => {
+			vm.model = { rating: 10 };
+		});
+
+		it("handle value should be the model value after changed", (done) => {
+			vm.$nextTick( () => {
+				let origin = input.querySelectorAll(".noUi-origin")[0];				
```

Comment:
Please use `querySelector` if you need the first element. Because `querySelectorArr()[0]` throw exception if the result is empty. However `querySelector` only return with null.